src/MonoTorrent/MonoTorrent.Client/Modes/HashingMode.cs

```diff
@@ -39,10 +39,9 @@ class HashingMode : Mode
 
         public override bool CanHashCheck => false;
 
-        bool Paused { get; set; }
         TaskCompletionSource<object> PausedCompletionSource { get; set; }
 
-        public override TorrentState State => Paused ? TorrentState.HashingPaused : TorrentState.Hashing;
+        public override TorrentState State => PausedCompletionSource.Task.IsCompleted ? TorrentState.Hashing : TorrentState.HashingPaused;
```

Comment:
I started off with a boolean and migrated to a TaskCompletionSource, and forgot to remove the boolean :)